collector/aspnet_aspnetapplications.go

```diff
@@ -0,0 +1,1239 @@
+// returns data points from Win32_PerfRawData_ASPNET_ASPNETApplications
+// <add link to documentation here> - Win32_PerfRawData_ASPNET_ASPNETApplications class
+package collector
+
+import (
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/log"
+)
+
+func init() {
+	Factories["aspnet_aspnetapplications"] = Newaspnet_aspnetapplicationsCollector
+}
+
+// A aspnet_aspnetapplicationsCollector is a Prometheus collector for WMI Win32_PerfRawData_ASPNET_ASPNETApplications metrics
+type aspnet_aspnetapplicationsCollector struct {
+	AnonymousRequests                     *prometheus.Desc
+	AnonymousRequestsPerSec               *prometheus.Desc
+	ApplicationLifetimeEvents             *prometheus.Desc
+	ApplicationLifetimeEventsPerSec       *prometheus.Desc
```

Comment:
Check if these two groups (AnonymousRequests and ApplicationLifetimeEvents) are actually different with/without PerSec? Typically we only want the counter of totals, not a pre-calculated rate.

Reply:
There are a bunch of more metrics below with two versions, with/without PerSec, I see.